429.html

```diff
@@ -2,19 +2,41 @@
 <html>
   <head>
     <link rel="stylesheet" type="text/css" href="main.css" />
-    <title>Error 429 - freeCodeCamp.org</title>
+    <title>Error 429 Class - freeCodeCamp.org</title>
   </head>
+
   <body>
     <div class="error-middle">
       <h2>
-        Sorry, you are unable to access this part of freeCodeCamp.
+        This part of freeCodeCamp is offline.
```

Comment:
Revert this. 429 is for someone making more requests than they should be and other changes.

Reply:
You can update the styles as you have on 500 and leave the YouTube link out. Sorry my bad when I asked you to make changes.